### DISCUSSION TITLE
mermaidJSのバージョンアップ（現在 mermaid version 8.14.0）

### DISCUSSION THREAD

**Discussion by SakakitaniJunya** [increments/qiita-discussions#475] [要望・アイデア・不具合]
mermaidJSのバージョンアップ希望しています。
mermaidJSは、10系までリリースされていて、8系ではtimelineなどのグラフが作成できません。
QiitaのmermaidJS（現在 mermaid version 8.14.0）

**Comment by tanaka-asami415** [comment]
@SakakitaniJunya 
ご不便をおかけして、申し訳ありません。
こちら対応予定ですので、お待ちいただけると幸いです🙏

**Reply by SakakitaniJunya** [reply]
ご連絡ありがとうございます。
いつ頃リリース予定になりますでしょうか

**Reply by tanaka-asami415** [reply]
@SakakitaniJunya 
お待たせして申し訳ありません。
今月着手予定ですので、もうしばらくお待ちいただけると幸いです🙇

**Reply by kyntk** [reply]
@SakakitaniJunya
お待たせいたしました :pray: 
Mermaidのバージョンアップを行いました :rocket: 
ぜひご活用ください！
https://qiita.com/release-notes/2023-07-26
